packages/connect-common/files/coins-eth.json

```diff
@@ -94,5 +94,24 @@
             "T2B1": "2.6.1"
         },
         "url": "https://ethereumclassic.org"
+    },
+    {
+        "blockchain_link": {
+            "type": "blockbook",
+            "url": ["https://matic1.trezor.io", "https://matic2.trezor.io"]
+        },
+        "chain": "matic",
+        "chain_id": 6193,
+        "name": "Polygon",
+        "shortcut": "MATIC",
+        "slip44": 1,
+        "support": {
+            "connect": true,
+            "suite": true,
+            "T1B1": "1.6.2",
+            "T2T1": "2.0.7",
+            "T2B1": "2.6.1"
+        },
+        "url": "https://polygon.technology"
```

Comment:
I am not 100% sure if this is needed here. It works event without it, doesn't it?